src/bin/cql-stress-cassandra-stress/operation/mod.rs

```diff
@@ -16,6 +23,121 @@ use scylla::{
 };
 pub use write::WriteOperationFactory;
 
+use crate::settings::CassandraStressSettings;
+use crate::stats::ShardedStats;
+
+use self::row_generator::RowGenerator;
+
+/// A specific CassandraStress operation.
+#[trait_variant::make(CassandraStressOperation: Send)]
+pub trait LocalCassandraStressOperation: Sync + Send + Sized {
+    type Factory: CassandraStressOperationFactory<Operation = Self>;
+
+    const DEFAULT_TABLE_NAME: &'static str = "standard1";
+    const DEFAULT_COUNTER_TABLE_NAME: &'static str = "counter1";
```

Comment:
Not sure that they should be associated constants. Are they supposed to be overriden? If not then I suggest making them module-level constants.